src/components/Header.js

```diff
@@ -55,19 +55,17 @@ class Header extends React.Component {
     })
     .then(res => res.json())
     .then(data => {
-      if (data.recipes) {
-        this.setState({recipes: data.recipes, totalCount: data.totalCount});
-        localStorage.setItem("recipes", LZString.compress(JSON.stringify(data.all)));
-      }
-    });
-    this.setState({
-      ingredients: [],
-      minTime: 0,
-      maxTime: 0,
-      createdBy: "",
-    });
+      this.setState({recipes: data.data, totalCount: data.totalCount});
+      this.setState({
+        ingredients: [], 
```

Comment:
what does this setState do? It seems to be just meaningless data?

Reply:
Originally, this was going to help with the searching part. After searching, I reset the values the user inputted, but I can reimplement/redesign this when I actually implement backend for searching and filtering recipes.